lib/guard/puma.rb

```diff
@@ -38,6 +40,8 @@ def start
     end
 
     def reload
+      return if Time.now - @last_restarted < options[:restart_timeout]
```

Comment:
Can you add parens to make the order of operations more clear?

Reply:
Yes, sure.